src/types.ts

```diff
@@ -14,3 +14,10 @@ export interface LanguageDictionary {
   [LocaleCodes.en]: CustomLocale
   [LocaleCodes.es]: CustomLocale
 }
+
+export enum SetupMessages {
+  runningWallet = 'Running wallet',
+  updatingWalletBackend = 'Updating wallet backend',
+  walletUpToDate = 'Wallet up to date',
+  loaded = 'Loaded',
+}
```

Comment:
I'd avoid using copies here for the UI. We should use this enum to choose the appropriate i18n key.
```suggestion
export enum SetupMessages {
  runningWallet,
  updatingWalletBackend,
  walletUpToDate,
  loaded,
}
```